README.md

```diff
@@ -169,6 +169,17 @@ $ make push
 
 If you're totally finished, go back to "Window 1" where the `make serve` command is still running, and hit CTRL-C to stop it.
 
+# Troubleshoot
```

Comment:
I'm trying to fully explain how to use the template, but this isn't a problem with mdbook or the template. This is a problem with the non-standard shell you've chosen to use.